onlinejudge/service/yukicoder.py

```diff
@@ -366,6 +371,12 @@ def from_url(cls, url: str) -> Optional['YukicoderProblem']:
             return cls()
         return None
 
+    @classmethod
+    def from_nothing(cls) -> Optional['YukicoderProblem']:
```

Comment:
This function is unnecessary.
Simply you can use `YukicoderProblem(problem_no=5555)` in your test.